requirements.txt

```diff
@@ -1,2 +1,3 @@
+beautifulsoup4==4.9.1
```

Comment:
Let's remove this requirement once #288 is merged. That will take care take care of installing plugin dependencies with the makefile target

Reply:
Or maybe use `-r plugins/summary/requirements.txt` ? @palnabarun Thoughts ?

Reply:
That's what the `make requirements` does.

Reply:
To summarize, the upcoming build process will install dependencies from `requirements` file from main app and plugins. So, I should not remove the `requirements` file from plugin directory. Did I understand it correctly?